### PR TITLE
feat: #7223 adds installed apps page

### PR DESCRIPTION
**Pull request by willmcvay**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #7223

### CODE REVIEW

File: packages/app-market/src/components/apps-installed/apps-installed-suggested.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  return 3
+}
+
+export const handleSortConfigs = (appsBrowseConfigState: AppsBrowseConfigCollection | null) => () => {
+  const featuredHeroApps =
+    appsBrowseConfigState?.data.filter((config) => config.configType === 'featuredHeroApps') ?? []
+  const heroApps = appsBrowseConfigState?.data.filter((config) => config.configType === 'heroApps') ?? []
+
+  const featuredHeroAppDetails = featuredHeroApps
+    .map(({ content, filters }) => {
```
Comment (bashleigh) [inline packages/app-market/src/components/apps-installed/apps-installed-suggested.tsx:31]: you can use reduce to minimize null returns 

```ts
featured = featuredHeroApps.reduce((array, { content, filters }) => {
  if (content?. imageUrl && Array.isArray(filters.id) && filters.id.length >= 1) {
     array.push({ imageUrl: content.imageUrl, id: filters.id[0], })
  }
}, [])
```

Reasons: in your method, there's 3 loops, the initial map, the flat and the filter func.

File: packages/app-market/src/components/apps-installed/apps-installed-suggested.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          imageUrl: content.imageUrl,
+          id: filters.id[0],
+        }
+      }
+    })
+    .flat()
+    .filter(isTruthy)
+
+  const heroAppDetails = heroApps
+    .map(({ content, filters }) => {
```
Comment (bashleigh) [inline packages/app-market/src/components/apps-installed/apps-installed-suggested.tsx:43]: same again, please see above

**Review by bashleigh — APPROVED** [review]
Approved but with a suggestion on minimizing loops

**Review by joshbalfour — APPROVED** [review]

File: packages/app-market/src/components/apps-installed/apps-installed-suggested.tsx (hunk 1 above)
Comment (willmcvay) [reply packages/app-market/src/components/apps-installed/apps-installed-suggested.tsx:31]: I tried that, firstly there are only 2 items in the array plus it's memoised so not expensive.

Also, none of the above typechecks, to actually make it work you have to type cast all over the place, set unhelpful defaults and things that will make the result less reliable.

In the end I chose reducing the risk of runtime errors over a small performance gain given we are only iterating over two items.

**Pull request merged**